.buildkite/release-builds.yml

```
@@ -10,6 +10,9 @@ agents:
 steps:
   - label: "Gradle Wrapper Validation"
     command: |
+      echo '--- :git: Checkout Release Branch'
+      .buildkite/commands/checkout-release-branch.sh "$RELEASE_VERSION"
+
```

Followup to the nitpick comment above, the reason I like it is that every call simplifies to

```suggestion
      .buildkite/commands/checkout-release-branch.sh "$RELEASE_VERSION"

```

Which I'll admit is not much, but ensures that if we change the string, it'll changes consistently across all usages.